project_app/components/Card.js

```diff
@@ -0,0 +1,45 @@
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Image from "next/image";
+import { Card, Typography } from "./styled";
+
+function CardGame() {
+  return (
+    <Card sx={{}}>
+      <CardMedia
```

Comment:
Hmm now Im thinking about it, can we rename the file and component name to `ReviewCard` ?

Reply:
Component renamed to `ReviewCard` @idiglove